resources/admin.py

```diff
@@ -35,17 +38,57 @@ class BoatTypeAdmin(TranslatableAdmin):
     pass
 
 
-class HarborAdmin(CustomTranslatableAdmin, admin.OSMGeoAdmin):
-    list_display = ("name",)
+class HarborAdmin(CustomTranslatableAdmin, admin.OSMGeoAdmin, HarborManager):
     ordering = ("translations__name",)
+    readonly_fields = (
+        "max_width",
+        "max_length",
+        "max_depth",
+    )
+
+    def max_width(self, obj):
+        return obj.max_width
+
+    def max_length(self, obj):
+        return obj.max_length
+
+    def max_depth(self, obj):
+        return obj.max_depth
+
+    max_width.short_description = _("Maximum allowed width")
+    max_width.admin_order_field = _("max_width")
```

Comment:
probably `admin_order_field` attributes do not need lazy translation wrappers?

Reply:
Whoops 😅 I got carried away by the copy-paste. Good catch.